content/en/real_user_monitoring/guide/enable-rum-shopify-store.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Enable RUM on your Shopify store
+kind: guide
+further_reading:
+- link: '/real_user_monitoring/guide/rum-for-product-analytics/'
+  tag: 'Documentation'
+  text: 'Use RUM & Session Replay for Product Analytics'
+- link: '/real_user_monitoring/guide/alerting-with-conversion-rates/'
+  tag: 'Documentation'
+  text: 'Alerting With Conversion Rates'
+---
+
+## Overview
+
+With the ever growing popularity of low-code e-commerce platforms such as Shopify, WooCommerce, or Squarespace, understanding how customers interact with your website — including which pages they visit, how much time they spend on your website, and which products are the most popular — is crucial to the success of your store. This data can be used to optimize your store and improve the overall customer experience.
```

Comment:
I would take out the examples and change it to "With the ever growing popularity of low-code e-commerce platforms, understanding how...